packages/design/src/Elevations.stories.mdx

```diff
@@ -104,3 +105,6 @@ they map to our low/base/high pattern.
 For mobile designs, the concept of elevation applies to both shadows and
 hierarchy of the element along the z-index. Android accepts a single "elevation"
 value, while iOS accepts more granular shadow values similar to CSS.
+
+When using the mobile tokens imported from `foundation.native`, the shadow
+tokens will be resolved to apply to either android or iOS devices automatically.
```

Comment:
I suppose not - I initially included it because the sentence before this: `Android accepts a single "elevation"
value, while iOS accepts more granular shadow values similar to CSS.` would make me question if I should be doing something different for iOS vs. android - or using these tokens at all.  I suppose this is slightly touched on at the top of the documentation where it says: `Our mobile components combine these two concepts in keeping with Android and iOS patterns.`.
I can definitely get rid of this sentence, but should we even keep the Mobile subheading it's within?

Reply:
i see, this makes sense actually